src/components/Disturber.tsx

```diff
@@ -80,11 +88,13 @@ export const Disturber = ({ navigation, publicJsonFile }: Props) => {
   if (!isVisible || !closestItem) return null;
 
   const {
+    aspectRatio,
+    autoplayInterval,
     backgroundColor,
     button,
     description,
     headline,
-    picture,
+    pictures,
     showButtonToClose = true,
     title
   } = closestItem;
```

Comment:
is there a specific reason why its written like this instead of below, or dosen't it make any difference?

const closestItem = {
    aspectRatio,
    autoplayInterval,
    backgroundColor,
    button,
    description,
    headline,
    picture,
    pictures,
    showButtonToClose = true,
    title
  };

Reply:
If we write the code with a method like you said, we create an object called closestItem in the code. Since we need the values in closestItem, the code must be in this way. Also in this way we can update the code over the server.